src/pytest_regressions/dataframe_regression.py

```diff
@@ -183,12 +185,12 @@ def _dump_fn(self, data_object, filename):
 
     def check(
         self,
-        data_frame,
-        basename=None,
-        fullpath=None,
-        tolerances=None,
-        default_tolerance=None,
-    ):
+        data_frame: Any,
```

Comment:
Is this `Any` because we don't want to enforce Pandas as a mandatory dependency?

Reply:
Exactly.